gitops/charts/postgres/postgres-test-application-values.yaml

```diff
@@ -129,7 +129,7 @@ instances:
       - "ReadWriteOnce"
       resources:
         requests:
-          storage: 1Gi
+          storage: 2Gi
```

Comment:
This sets the data PVC to 2Gi to match what was provisioned previously. The PVC may have been expanded outside the CrunchyDB resource which would can cause events to be raised in the cluster.